src/main/resources/assets/schemas/Institution.yaml

```diff
@@ -6,6 +6,31 @@ properties:
   name:
     type: string
     description: Name of institution
+  itDirectorName:
+    type: string
+    description: Name of IT Director
+  itDirectorEmail:
+    type: string
+    description: Email of IT Director
+  institutionUrl:
+    type: string
+    description: Link to main page of institution
+  dunsNumber:
+    type: string
+    description: '?'
```

Comment:
unsure what this should be

Reply:
This can be something as simple as '"9-digit identification number provided by Duns & Bradstreet"

Reply:
Yeah, I think keeping it simple would be good "A unique nine-digit
identification number for businesses provided by Dun & Bradstreet."
Here's D&B's own explanation of it:
https://www.dnb.com/ca-en/duns-number.html